pkg/console/operator/status.go

```diff
@@ -235,96 +232,6 @@ func (c *consoleOperator) ConditionResourceSyncDegraded(operatorConfig *operator
 	return operatorConfig
 }
 
-func (c *consoleOperator) ConditionsManagementStateUnmanaged(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type: operatorsv1.OperatorStatusTypeAvailable,
-		// See https://github.com/openshift/api/pull/206
-		// While the ConditionUnknown state seems to be the correct fit, the current understanding is that
-		// If the operator is fulfilling the user's desired state, set Available:true
-		// Status:             operatorsv1.ConditionUnknown,
-		Status:             operatorsv1.ConditionTrue,
-		Reason:             reasonUnmanaged,
-		Message:            "The operator is in an unmanaged state, therefore its availability is unknown.",
-		LastTransitionTime: metav1.Now(),
-	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeProgressing,
-		Status:             operatorsv1.ConditionFalse,
-		Reason:             reasonUnmanaged,
-		Message:            "The operator is in an unmanaged state, therefore no changes are being applied.",
-		LastTransitionTime: metav1.Now(),
-	})
-	v1helpers.SetOperatorCondition(&operatorConfig.Status.Conditions, operatorsv1.OperatorCondition{
-		Type:               operatorsv1.OperatorStatusTypeDegraded,
-		Status:             operatorsv1.ConditionFalse,
-		Reason:             reasonUnmanaged,
-		Message:            "The operator is in an unmanaged state, therefore no operator actions are degraded.",
-		LastTransitionTime: metav1.Now(),
-	})
-
-	return operatorConfig
-}
-
-func (c *consoleOperator) ConditionsManagementStateRemoved(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
```

Comment:
Need to take care of https://github.com/openshift/console-operator/blob/master/pkg/console/operator/operator.go#L228